react/CozyDialogs/ConfirmDialog.jsx

```diff
@@ -40,11 +41,13 @@ const ConfirmDialog = props => {
           onClick={onBackOrClose}
         />
       )}
-      <DialogContent>
+      <DialogContent {...dialogContentProps}>
         <div className="dialogContentInner withFluidActions">
-          <div className="dialogTitleFluidContainer">
-            <DialogTitle {...dialogTitleProps}>{title}</DialogTitle>
-          </div>
+          {title && (
+            <div className="dialogTitleFluidContainer">
+              <DialogTitle {...dialogTitleProps}>{title}</DialogTitle>
+            </div>
+          )}
```

Comment:
Are we sure it does break anything? Or at least have we checked that the display if the same (before & after) without title ?